modules/ztp-setting-managed-bare-metal-host-kernel-parameters-manual-workflow_gitops_workflow.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-sites.adoc
+
+[id="setting-managed-bare-metal-host-kernel-parameters_{context}"]
+= Setting managed bare-metal host kernel parameters
+
+Configure kernel parameters for managed bare-metal hosts to customize the boot process. This is useful for clusters with specific environmental requirements. For example, configure the `rd.net.timeout.carrier` kernel parameter to facilitate static networking or to receive a DHCP address before downloading the root file system. 
+
+[NOTE]
+====
+In {product-title} {product-version}, you can only add kernel parameters. You can not replace or delete kernel parameters.
+====
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (oc).
+* You have logged in to the hub cluster as a user with cluster-admin privileges.
+
+.Procedure
+
+. Create the `InfraEnv` CR and edit the `spec.kernelArguments` specification to configure kernel parameters.
+
+.. Save the following YAML in an `InfraEnv-example.yaml` file, replacing the variables to match the details of your cluster:
```

Comment:
It might be helpful to indicate that this is a template so that the customer keeps the {{....}} syntax intact